docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md

```diff
@@ -0,0 +1,87 @@
+---
+id: using-existing-elasticsearch
+title: "Using existing Elasticsearch"
+description: "Learn how to use an existing elasticsearch instance in Camunda 8 Self-Managed deployment."
+---
+
+Camunda 8 Self-Managed has two different types of components: Camunda components (Operate, Optimize, Tasklist, etc.) and non-Camunda, dependency components (such as Keycloak and Elasticsearch). For more details, review the [architecture](../../../platform-architecture/overview.md) documentation for more information on the different types of applications.
+
+This guide steps through using an existing elasticsearch instance. By default, [Helm chart deployment](../deploy.md) creates a new elassticsearch instance, but it's possible to use an existing elasticsearch instance either inside the same Kubernetes cluster or outside of it.
+
+## Preparation
+
+### Connecting to self managed elasticsearch
+
+<!-- You must be aware of the username and password needed to connect to your Elasticsearch cluster. -->
+<!-- You must be aware of the hostname of the elasticsearch cluster. -->
+
+You must be aware of the following information relating to your self managed elasticsearch cluster:
+
+1. protocol, host, port
```

Comment:
Let's capitalize the words here.

Reply:
done

Reply:
The same for the next line `username and password`.

Reply:
done